### PR TITLE
MGMT-11926: vsphere: create machines with HW 15

### PR DESCRIPTION
**Pull request by vrutkovs**
Hardware version 15 is required for CSI tests to pass

### CODE REVIEW

**Comment by vrutkovs** [comment]
/cc @eliorerz 
/hold

for rehearsals

**Review by osherdp — APPROVED** [review]
:pizza:

File: terraform_files/vsphere/main.tf [hunk 1]
```diff
@@ -91,6 +91,7 @@ resource "vsphere_virtual_machine" "master" {
   guest_id                    = "coreos64Guest"
   folder                      = var.vsphere_folder != "" ? "${var.vsphere_parent_folder}/${local.folder}" : vsphere_folder.folder[0].path
   enable_disk_uuid            = "true"
+  hardware_version            = 15
```
Comment (romfreiman) [inline terraform_files/vsphere/main.tf:94]: what's the default?

Comment (vrutkovs) [reply terraform_files/vsphere/main.tf:94]: That is baked in OVA which we use, so we start with HW 14, run 4.11 CSI tests and these complain that you need at least 15 - see `[sig-arch][Early] Managed cluster should start all core operators` test message in this [test run](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-assisted-test-infra-master-e2e-vsphere-assisted-periodic/1572012854410743808)

**Comment by vrutkovs** [comment]
UMN is known to be broken, unholding

/hold cancel

**Pull request merged**